cmd/apiserver-boot/boot/build/build_resource_config.go

```diff
@@ -283,7 +283,6 @@ spec:
   version: {{ $api.Version }}
   group: {{ $api.Group }}.{{ $config.Domain }}
   groupPriorityMinimum: 2000
-  priority: 200
```

Comment:
set the version priority to 1 please since that 0 priority will fail validation

Reply:
note that version priority should range in (0, 1000)

Reply:
it is 10 now. you should check the whole file